src/styles.css

```diff
@@ -0,0 +1,248 @@
+*,
+*::after,
+*::before {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+}
+
+:root {
+    overflow-x: hidden;
+}
+
+.pageBackground {
+    position: absolute;
+    top: 0;
+    width: 100vw;
+    height: 100vh;
+    transition: 300ms 100ms linear;
+    z-index: -10;
+}
+
+.pageBackground--scale {
+    background: black;
+    opacity: 0.7;
+    z-index: 20;
+}
+
+.screen-readers {
+    position: absolute;
+    right: 2400px;
+}
+
+.header {
+    font-size: 1.5em;
+    text-align: center;
+    color: #cbebff;
+}
+
+.wrapper {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0 10%;
+    position: relative;
+}
+
+.header--main {
+    font-size: 2.5em;
+    color: #0014a6;
+}
+
+.arrows-switch {
+    position: absolute;
+    left: 10%;
+    padding: 10px;
+    display: flex;
+    align-items: center;
+}
+
+.arrows-switch__container {
+    width: 100px;
+    height: 60px;
+    border-radius: 30px;
+    background: #a63f00;
+    padding: 10px;
+    cursor: pointer;
+    display: inline-block;
+}
+
+.arrows-switch__paragraph {
+    display: inline;
+    font-size: 2em;
+    margin: 5px;
+    color: #0014a6;
+}
+.arrows-switch__point {
+    height: 40px;
+    border-radius: 50%;
+    width: 40px;
+    background: white;
+    transition: 200ms 50ms linear;
+}
+
+.point--active {
+    transform: translateX(40px);
+}
+
+.button {
+    border: 0;
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    background: transparent;
+    cursor: pointer;
+    z-index: 10;
+    opacity: 1;
+    transition: 200ms 50ms linear;
+}
+
+.button--hidden {
+    opacity: 0;
+}
+
+.button--left {
+    left: 0;
+}
+
+.button--right {
+    right: 0;
+}
+
+.container__svg {
+    width: 50px !important;
+    height: 50px !important;
+}
```

Comment:
o jak przeniesiesz width i height z svg tu, to nie będziesz potrzebował `!important`